public/css/layers-panel.css

```
@@ -0,0 +1,1169 @@
+  transition: background-color 0.2s;
+}
+
+@media (max-width: 768px) {
+  .layer-btn {
+    width: 28px;
+    height: 28px;
+  }
+  .layer-btn i {
+    font-size: 14px;
+  }
+}/*# sourceMappingURL=layers-panel.css.map */
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)